src/app/hearing-part/components/listing-note-list/listing-note-list.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
+import { NoteListComponent } from '../../../notes/components/notes-list/note-list.component';
+import { Note } from '../../../notes/models/note.model';
+import { ListingCreateNotesConfiguration } from '../../models/listing-create-notes-configuration.model';
+import { getNoteViewModel, NoteViewmodel } from '../../../notes/models/note.viewmodel';
+import { NotesPreparerService } from '../../../notes/services/notes-preparer.service';
+
+@Component({
+  selector: 'app-listing-note-list',
+  templateUrl: './listing-note-list.component.html',
+  styleUrls: ['./listing-note-list.component.scss']
+})
+export class ListingNoteListComponent implements OnInit {
+  @ViewChild('notesList') noteList: NoteListComponent;
+  @ViewChild('freeTextNotesList') freeTextNoteList: NoteListComponent;
+
+  public _notes: Note[];
+  public noteViewModels: NoteViewmodel[] = [];
+  public freeTextNoteViewModels: NoteViewmodel[] = [];
+
+  @Input() set notes(value: Note[]) {
+      this._notes = value;
+  }
+
+  @Input()
+  public entityId: string;
+
+  ngOnInit() {
+      this.initiateNotes(this._notes);
+  }
+
+  constructor(readonly listingNotesConfig: ListingCreateNotesConfiguration,
+            public notePreparerService: NotesPreparerService) { }
+
+  initiateNotes(notes: Note[]) {
```

Comment:
It is a presentational component because it does not interfere with external services or data. There is lots of transformations going on but they only apply to functional and UI behaviour. There might be a way to simplify the logic, but that's all I could do given the time